docs/modules/mapstore/pages/configuring-a-generic-mapstore.adoc

```diff
@@ -0,0 +1,383 @@
+= Configuring a Map with the Generic MapStore
+:description: You can configure a map with the generic MapStore to use it as a cache with an external data store.
```

Comment:
This is actually a beta feature in 5.2. Should be marked as such. (depends on JDBC SQL Connector which is also beta).